ocs_ci/ocs/ui/views.py

```diff
@@ -18,13 +18,17 @@
         'a[title="Log in with my_htpasswd_provider"]',
         By.CSS_SELECTOR,
     ),
-    "skip_tour": (By.CSS_SELECTOR, 'button[data-test="tour-step-footer-secondary"]'),
+    "skip_tour": ('button[data-test="tour-step-footer-secondary"]', By.CSS_SELECTOR),
 }
 
 login_4_11 = {
     "ocp_page": "Overview · Red Hat OpenShift",
     "login_page_title": "Log in · Red Hat OpenShift",
 }
+# Bug opened in Jira https://issues.redhat.com/browse/OCPBUGS-15419. Tmp solution to check locators
+login_4_14 = {
```

Comment:
Can you move it to https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/views.py
?